src/System.Net.Requests/src/System/Net/HttpVersion.cs

```diff
@@ -0,0 +1,28 @@
+//------------------------------------------------------------------------------
+// <copyright file="HttpVersion.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>                                                                
+//------------------------------------------------------------------------------
+
+namespace System.Net {
+
+    /// <devdoc>
+    ///    <para>
+    ///       Defines the HTTP version number supported by the <see cref='System.Net.HttpWebRequest'/> and
+    ///    <see cref='System.Net.HttpWebResponse'/> classes.
+    ///    </para>
+    /// </devdoc>
+    public class HttpVersion {
```

Comment:
I agree that maybe HttpVersion should be put into something like System.Net.Primitives.

And while we would also like to unify it with the one in Common, that one has two additional values in it.  And we can't put those into the public HttpVersion without moving changing the public API and dealing with getting into .NET Framework vnext (4.6.3).

cc: @ericstj @weshaggard @terrajobst - thoughts on this?